### ISSUE TITLE
How can I use it on Android?

### ISSUE THREAD

**Issue by yijunwu** [ymzis69/HybridSORT#20]
Suppose I have detector running on Android already, how can I run HybridSORT on top of it? Any suggestion on approaches?

**Comment by ymzis69** [comment]
Hello, as long as you can run the detector on Android, you can certainly implement the tracking algorithm based on the detection results. The specific deployment can be achieved through algorithms such as C++. You can refer to code repositories like this one: https://github.com/yaoyi30/ByteTrack_ncnn_Android

**Comment by yijunwu** [comment]
> Hello, as long as you can run the detector on Android, you can certainly implement the tracking algorithm based on the detection results. The specific deployment can be achieved through algorithms such as C++. You can refer to code repositories like this one: https://github.com/yaoyi30/ByteTrack_ncnn_Android

Thank you for the reply. So it means that I need to manually convert the python code in this repo to C++, right?

**Comment by ymzis69** [comment]
Yes, there is currently no ready-made C++ version of HybridSORT. The main code that needs to be converted is located in trackers/hybrid_sort_tracker/.

**Comment by yijunwu** [comment]
Thank you very much for the answer.
